lib/utils.js

```diff
@@ -13,6 +13,24 @@ const matter = require('gray-matter')
 // Local dependencies
 const config = require('../app/config.js')
 
+// Are we running on Glitch.com?
+const onGlitch = function () {
+  // there isn't an official way to check, but this was recommended
+  // https://support.glitch.com/t/detect-if-app-is-running-on-glitch/3120
+  return Boolean(process.env.PROJECT_REMIX_CHAIN)
+}
+
+exports.onGlitch = onGlitch
+
+// Are we running in production, or in development?
+exports.inProduction = function () {
+  const env = (process.env.NODE_ENV || 'development').toLowerCase()
+  return Boolean(
```

Comment:
is there value in `Boolean` here? Will it do the same without it?

Reply:
Not using booleans any more, so can consider this resolved 😂